lib/content-services/src/lib/common/services/nodes-api.service.ts

```diff
@@ -246,6 +257,14 @@ export class NodesApiService {
         return this.createNodeInsideRoot(name || this.randomNodeName(), nodeType, properties, path);
     }
 
+    initiateFolderSizeCalculation(nodeId: string): Observable<JobIdBodyEntry> {
```

Comment:
Missing docs

Reply:
Please also update `nodes-api.service.md`

Reply:
Added documentation and updated nodes-api.service.md as well